src/server/database/migrations/20180423122353_create-tokens.js

```diff
@@ -0,0 +1,26 @@
+export const up = async knex => {
+  await knex.schema.createTable('tokens', table => {
+    table
+      .increments('id')
+      .unsigned()
+      .primary()
+    table
+      .string('token')
+      .unique()
+      .notNull()
+    table
+      .integer('userId')
+      .unsigned()
+      .notNull()
+      .unique()
+    table.foreign('userId').references('users.id')
+    table
+      .timestamp('createdAt')
+      .notNull()
+      .defaultTo(knex.fn.now())

```

Comment:
We don't have a proper way to handle "touching" resource (updating this column every time a resource is fetched, not modified), and I'm not sure it's a priority right now. Although it could help us save user logins, we should have logging for that.